diff --git a/routers/router.go b/routers/router.go
index ef0e02a..d284273 100644
--- a/routers/router.go
+++ b/routers/router.go
@@ -14,6 +14,7 @@ var (
 )
 
 func init() {
+	beego.LoadAppConfig("ini", "/etc/nekobox/nekobox.conf")
 	beego.InsertFilter("*", beego.BeforeExec, func(c *context.Context) {
 		c.Input.Data()["title"] = beego.AppConfig.String("title")
 		c.Input.Data()["icp"] = beego.AppConfig.String("icp")
